src/Components/MapComponent/MapComponent.jsx

```diff
@@ -44,21 +43,20 @@ import * as actions from '../../store/actions';
  * @property {string} routingUrl The API routing url to be used for navigation.
  * @property {string} currentMot The current selected mot by user, example 'bus'.
  * @property {Object} currentStopsGeoJSON The current stops defined by user in geojson format inside a dictionary, key is the stop index(order) and the value is the geoJSON itself.
- * @property {function} onShowNotification A store action that can be dispatched, takes the notification message and type as arguments.
- * @property {function} onSetClickLocation A store action that can be dispatched, takes the clicked location on map array of [long,lat] and stores it in the store.
+ * @property {function} dispatchShowNotification A store action that can be dispatched, takes the notification message and type as arguments.
+ * @property {function} dispatchSetClickLocation A store action that can be dispatched, takes the clicked location on map array of [long,lat] and stores it in the store.
  * @category Props
  */
 
 let abortController = new AbortController();
-const zoom = 6;
 let cbKey = null;
 
 /**
  * The only true map that shows inside the application.
  * @category Map
  */
 class MapComponent extends PureComponent {
-  static getExtentCenter = extent => {
+  static getExtentCenter = (extent) => {
```

Comment:
Don't hesitate to move static function inside an utils file. Make the code smaller and easier to test

Reply:
These static functions were all useless I just noticed, must be some relics from a while ago.
I nevertheless created a utils folder and moved separated some of the util functions into separate files